scripts/integration/docker-compose.datadog-traces.yml

```diff
@@ -1,91 +0,0 @@
-version: "3"
-
-services:
-  datadog-trace-agent:
```

Comment:
@neuronull we still need both this and `datadog-trace-agent-to-vector` for integration tests to pass?

Reply:
I have a feeling reverting the line in question here:

https://github.com/vectordotdev/vector/pull/16130#discussion_r1087048606

, and the change to [scripts/integration/datadog-agent/test.yaml](https://github.com/vectordotdev/vector/pull/16130/files#diff-5df4230c15b5bb57516aff1965f11a5d726bd5031414d4e5459e822c8eb3acdb)

, is what is needed.

I think the deletion of this file you pointed to is just part of moving it over to vdev, if I'm not mistaken?